exp/ingest/ledgerbackend/database_backend.go

```diff
@@ -46,6 +46,9 @@ func (dbb *DatabaseBackend) GetLatestLedgerSequence() (uint32, error) {
 	if err != nil {
 		return 0, errors.Wrap(err, "couldn't select ledger sequence")
 	}
+	if len(ledger) == 0 {
+		return 0, errors.New("No ledgers found in ledgerheaders table")
```

Comment:
Errors should not be capatlized for these reasons: https://github.com/golang/go/wiki/CodeReviewComments#error-strings
```suggestion
		return 0, errors.New("no ledgers found in ledgerheaders table")
```